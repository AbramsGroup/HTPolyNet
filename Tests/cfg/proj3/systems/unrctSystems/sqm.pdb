ATOM      1  O   FDE     1       0.987   2.392   0.463  1.00  0.00           O
ATOM      2  C   FDE     1       0.273   2.712   1.617  1.00  0.00           C
ATOM      3  C3  FDE     1      -0.042   1.566   2.318  1.00  0.00           C
ATOM      4  C4  FDE     1       1.113   0.995   0.449  1.00  0.00           C
ATOM      5  C5  FDE     1       0.502   0.457   1.564  1.00  0.00           C
ATOM      6  C6  FDE     1       1.804   0.411  -0.727  1.00  0.00           C
ATOM      7  N   FDE     1       3.233   0.201  -0.519  1.00  0.00           N
ATOM      8  H   FDE     1       1.608   1.077  -1.621  1.00  0.00           H
ATOM      9  H1  FDE     1       1.333  -0.597  -0.932  1.00  0.00           H
ATOM     10  C7  FDE     1       3.983   1.445  -0.432  1.00  0.00           C
ATOM     11  C8  FDE     1       3.800  -0.727  -1.488  1.00  0.00           C
ATOM     12  C9  FDE     1       3.449  -2.202  -1.202  1.00  0.00           C
ATOM     13  H2  FDE     1       4.922  -0.635  -1.449  1.00  0.00           H
ATOM     14  H3  FDE     1       3.468  -0.491  -2.541  1.00  0.00           H
ATOM     15  C1  FDE     1       4.510  -3.114  -1.792  1.00  0.00           C
ATOM     16  H4  FDE     1       4.625  -2.922  -2.885  1.00  0.00           H
ATOM     17  H5  FDE     1       4.220  -4.183  -1.650  1.00  0.00           H
ATOM     18  H6  FDE     1       5.490  -2.941  -1.288  1.00  0.00           H
ATOM     19  O1  FDE     1       2.185  -2.438  -1.815  1.00  0.00           O
ATOM     20  H7  FDE     1       3.361  -2.388  -0.094  1.00  0.00           H
ATOM     21  C10 FDE     1       5.260   1.331   0.428  1.00  0.00           C
ATOM     22  H8  FDE     1       3.324   2.234   0.030  1.00  0.00           H
ATOM     23  H9  FDE     1       4.293   1.821  -1.452  1.00  0.00           H
ATOM     24  O2  FDE     1       6.089   2.452   0.159  1.00  0.00           O
ATOM     25  C2  FDE     1       4.934   1.228   1.904  1.00  0.00           C
ATOM     26  H10 FDE     1       5.889   0.454   0.105  1.00  0.00           H
ATOM     27  H11 FDE     1       4.337   2.111   2.237  1.00  0.00           H
ATOM     28  H12 FDE     1       5.877   1.194   2.500  1.00  0.00           H
ATOM     29  H13 FDE     1       4.342   0.303   2.100  1.00  0.00           H
ATOM     30  H14 FDE     1       5.628   3.236   0.482  1.00  0.00           H
ATOM     31  H15 FDE     1       1.983  -3.372  -1.692  1.00  0.00           H
ATOM     32  H16 FDE     1       0.094   3.774   1.754  1.00  0.00           H
ATOM     33  H17 FDE     1      -0.590   1.488   3.252  1.00  0.00           H
ATOM     34  H18 FDE     1       0.433  -0.592   1.837  1.00  0.00           H
