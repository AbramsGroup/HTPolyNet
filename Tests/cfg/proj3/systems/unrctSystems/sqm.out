            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 43

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       79
| QMMM:    diag iterations used for timing =       15
| QMMM:
| QMMM:              Internal diag routine = 0.015410 seconds
| QMMM:                 Dspev diag routine = 0.016100 seconds
| QMMM:                Dspevd diag routine = 0.011684 seconds
| QMMM:                Dspevx diag routine = 0.061052 seconds
| QMMM:                 Dsyev diag routine = 0.017472 seconds
| QMMM:                Dsyevd diag routine = 0.015085 seconds
| QMMM:                Dsyevr diag routine = 0.023595 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.008659 seconds
| QMMM:
| QMMM: Using dspevd routine (diag_routine=3).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      O       -0.1470    1.3670   -0.2320
  QMMM:     2        2      C       -0.9000    1.0270    0.7890
  QMMM:     3        3      C       -0.1260    0.4150    1.7530
  QMMM:     4        4      C        1.0910    1.0020    0.0150
  QMMM:     5        5      C        1.1580    0.4000    1.2590
  QMMM:     6        6      C        2.2230    1.2420   -0.9450
  QMMM:     7        7      N        3.5020    0.6910   -0.4220
  QMMM:     8        8      H        2.2590    2.3440   -1.0860
  QMMM:     9        9      H        1.9750    0.8210   -1.9440
  QMMM:    10       10      C        4.5900    1.6930   -0.5680
  QMMM:    11       11      C        3.8610   -0.6060   -1.0640
  QMMM:    12       12      C        2.9150   -1.7740   -0.7000
  QMMM:    13       13      H        4.8570   -0.9270   -0.7060
  QMMM:    14       14      H        3.9310   -0.4910   -2.1700
  QMMM:    15       15      C        3.4590   -3.0940   -1.2540
  QMMM:    16       16      H        3.5280   -3.0510   -2.3620
  QMMM:    17       17      H        2.7840   -3.9290   -0.9700
  QMMM:    18       18      H        4.4670   -3.3040   -0.8370
  QMMM:    19       19      O        1.6330   -1.5620   -1.2310
  QMMM:    20       20      H        2.8730   -1.8470    0.4110
  QMMM:    21       21      C        5.8390    1.3870    0.2950
  QMMM:    22       22      H        4.2260    2.6860   -0.2170
  QMMM:    23       23      H        4.8800    1.8000   -1.6380
  QMMM:    24       24      O        6.7970    2.4030    0.1310
  QMMM:    25       25      C        5.4880    1.2210    1.7790
  QMMM:    26       26      H        6.3210    0.4490   -0.0420
  QMMM:    27       27      H        5.0220    2.1470    2.1790
  QMMM:    28       28      H        6.4100    1.0040    2.3610
  QMMM:    29       29      H        4.7820    0.3750    1.9190
  QMMM:    30       30      H        6.4130    3.2470    0.4890
  QMMM:    31       31      H        0.9990   -2.0890   -0.6780
  QMMM:    32       32      H       -1.9650    1.2070    0.8510
  QMMM:    33       33      H       -0.4610    0.0270    2.7050
  QMMM:    34       34      H        2.0320   -0.0020    1.7530

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -101.9723 kcal/mol        0.6115 kcal/(mol*A)
xmin    20     -102.9684 kcal/mol        0.7056 kcal/(mol*A)
xmin    30     -103.4404 kcal/mol        1.0675 kcal/(mol*A)
xmin    40     -103.5727 kcal/mol        0.2084 kcal/(mol*A)
xmin    50     -103.9020 kcal/mol        0.3629 kcal/(mol*A)
xmin    60     -104.3852 kcal/mol        0.3872 kcal/(mol*A)
xmin    70     -104.6758 kcal/mol        0.1518 kcal/(mol*A)
xmin    80     -104.7625 kcal/mol        0.0951 kcal/(mol*A)
xmin    90     -104.7878 kcal/mol        0.0826 kcal/(mol*A)
xmin   100     -104.8024 kcal/mol        0.0421 kcal/(mol*A)
xmin   110     -104.8240 kcal/mol        0.2421 kcal/(mol*A)
xmin   120     -104.8456 kcal/mol        0.0742 kcal/(mol*A)
xmin   130     -104.8719 kcal/mol        0.0411 kcal/(mol*A)
xmin   140     -104.9334 kcal/mol        0.3769 kcal/(mol*A)
xmin   150     -105.0355 kcal/mol        0.0786 kcal/(mol*A)
xmin   160     -105.0831 kcal/mol        0.0372 kcal/(mol*A)
xmin   170     -105.1165 kcal/mol        0.0309 kcal/(mol*A)
xmin   180     -105.1253 kcal/mol        0.0159 kcal/(mol*A)
xmin   190     -105.1262 kcal/mol        0.0096 kcal/(mol*A)
xmin   200     -105.1279 kcal/mol        0.0241 kcal/(mol*A)
xmin   210     -105.1280 kcal/mol        0.0024 kcal/(mol*A)
xmin   220     -105.1281 kcal/mol        0.0041 kcal/(mol*A)
xmin   230     -105.1281 kcal/mol        0.0059 kcal/(mol*A)
xmin   240     -105.1281 kcal/mol        0.0009 kcal/(mol*A)
xmin   250     -105.1281 kcal/mol        0.0007 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -41.6050  -39.6825  -38.5848  -36.9808  -33.2613  -32.4656  -31.9258  -30.4443
 -27.6688  -26.8946  -24.2840  -22.3799  -22.1144  -21.6370  -20.7270  -18.4405
 -17.7466  -17.5387  -16.9693  -16.6568  -16.2196  -15.7218  -15.4428  -15.1256
 -14.6747  -14.4129  -13.9572  -13.7763  -13.7402  -13.5512  -13.4171  -12.9622
 -12.8501  -12.6432  -12.5888  -12.4776  -11.9455  -11.4939  -11.0994  -11.0395
 -10.5547   -9.3332   -9.0596    0.6651    1.6855    2.4530    2.8086    3.3153
   3.5412    3.6109    3.7107    3.8632    3.8778    3.9370    4.0216    4.0534
   4.0868    4.1357    4.2368    4.3302    4.4287    4.4759    4.5662    4.6179
   4.6604    4.7832    4.8380    4.9016    4.9038    4.9956    5.0482    5.1501
   5.2430    5.7132    5.9797    6.0524    6.1240    6.2039    6.4390

 Heat of formation   =       -105.12813407 kcal/mol  (       -4.55869798 eV)

 Total SCF energy    =     -65443.88152522 kcal/mol  (    -2837.85965592 eV)
 Electronic energy   =    -395442.36641069 kcal/mol  (   -17147.66776856 eV)
 Core-core repulsion =     329998.48488547 kcal/mol  (    14309.80811263 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      O                 -0.115
     2      C                 -0.098
     3      C                 -0.200
     4      C                 -0.041
     5      C                 -0.179
     6      C                 -0.010
     7      N                 -0.260
     8      H                  0.078
     9      H                  0.136
    10      C                 -0.106
    11      C                 -0.062
    12      C                  0.023
    13      H                  0.097
    14      H                  0.078
    15      C                 -0.250
    16      H                  0.088
    17      H                  0.076
    18      H                  0.087
    19      O                 -0.338
    20      H                  0.091
    21      C                  0.028
    22      H                  0.102
    23      H                  0.071
    24      O                 -0.329
    25      C                 -0.240
    26      H                  0.112
    27      H                  0.074
    28      H                  0.081
    29      H                  0.100
    30      H                  0.196
    31      H                  0.200
    32      H                  0.187
    33      H                  0.160
    34      H                  0.162
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -0.645   -1.345    0.623    1.617
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      O        0.9865    2.3917    0.4630
  QMMM:     2        2      C        0.2725    2.7119    1.6173
  QMMM:     3        3      C       -0.0422    1.5657    2.3177
  QMMM:     4        4      C        1.1130    0.9946    0.4487
  QMMM:     5        5      C        0.5018    0.4568    1.5642
  QMMM:     6        6      C        1.8039    0.4107   -0.7274
  QMMM:     7        7      N        3.2333    0.2011   -0.5188
  QMMM:     8        8      H        1.6079    1.0768   -1.6208
  QMMM:     9        9      H        1.3328   -0.5968   -0.9318
  QMMM:    10       10      C        3.9830    1.4450   -0.4325
  QMMM:    11       11      C        3.8003   -0.7273   -1.4876
  QMMM:    12       12      C        3.4492   -2.2020   -1.2017
  QMMM:    13       13      H        4.9224   -0.6353   -1.4488
  QMMM:    14       14      H        3.4682   -0.4912   -2.5408
  QMMM:    15       15      C        4.5102   -3.1141   -1.7916
  QMMM:    16       16      H        4.6252   -2.9217   -2.8850
  QMMM:    17       17      H        4.2197   -4.1829   -1.6498
  QMMM:    18       18      H        5.4903   -2.9409   -1.2878
  QMMM:    19       19      O        2.1853   -2.4376   -1.8152
  QMMM:    20       20      H        3.3607   -2.3882   -0.0943
  QMMM:    21       21      C        5.2598    1.3311    0.4283
  QMMM:    22       22      H        3.3236    2.2343    0.0303
  QMMM:    23       23      H        4.2928    1.8212   -1.4518
  QMMM:    24       24      O        6.0893    2.4523    0.1588
  QMMM:    25       25      C        4.9337    1.2285    1.9039
  QMMM:    26       26      H        5.8892    0.4541    0.1054
  QMMM:    27       27      H        4.3366    2.1112    2.2375
  QMMM:    28       28      H        5.8769    1.1936    2.5002
  QMMM:    29       29      H        4.3422    0.3033    2.0999
  QMMM:    30       30      H        5.6283    3.2355    0.4819
  QMMM:    31       31      H        1.9828   -3.3719   -1.6918
  QMMM:    32       32      H        0.0937    3.7737    1.7541
  QMMM:    33       33      H       -0.5899    1.4877    3.2521
  QMMM:    34       34      H        0.4333   -0.5920    1.8370

           --------- Calculation Completed ----------

